android/java/org/chromium/chrome/browser/brave_news/BraveNewsControllerFactory.java

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.brave_news;
+
+import org.chromium.base.annotations.CalledByNative;
+import org.chromium.base.annotations.JNINamespace;
+import org.chromium.base.annotations.NativeMethods;
+import org.chromium.brave_news.mojom.BraveNewsController;
+import org.chromium.mojo.bindings.ConnectionErrorHandler;
+import org.chromium.mojo.bindings.Interface;
+import org.chromium.mojo.bindings.Interface.Proxy.Handler;
+import org.chromium.mojo.system.MessagePipeHandle;
+import org.chromium.mojo.system.MojoException;
+import org.chromium.mojo.system.impl.CoreImpl;
+
+@JNINamespace("chrome::android")
+public class BraveNewsControllerFactory {
+    private static final Object lock = new Object();
+    private static BraveNewsControllerFactory instance;
```

Comment:
`sInstance`, pls see https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/java/java.md and https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions